subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -447,28 +447,55 @@ void *radio_pkt_decrypt_get(void)
 #endif
 
 #if !defined(CONFIG_BT_CTLR_TIFS_HW)
+
+#define SW_SWITCH_PREV_RX             0
+#define SW_SWITCH_NEXT_RX             0
+#define SW_SWITCH_PREV_TX             1
+#define SW_SWITCH_NEXT_TX             1
+#define SW_SWITCH_PREV_PHY_1M         0
```

Comment:
better use `PHY_LEGACY` or even better `PHY_1M` instead since this is actual value passed from LLL

Reply:
Same here, macros are used as readable string (instead of using defines) which would convey the parameters purpose. Otherwise, the call at line 607 will look like:
```
sw_switch(SW_SWITCH_DIR_TX, SW_SWITCH_DIR_RX,
		  PHY_1M, SW_SWITCH_PREV_FLAGS_DONTCARE,
		  phy_rx, SW_SWITCH_NEXT_FLAGS_DONTCARE);
```